packages/lime-debug/Makefile

```diff
@@ -23,8 +23,9 @@ define Package/$(PKG_NAME)
 	MAINTAINER:=Gioacchino Mazzurco <gio@eigenlab.org>
 	URL:=http://libremesh.org
 	DEPENDS:=+PACKAGE_lime-proto-batadv:batctl \
-		+busybox +ethtool +iwinfo +iw +mtr +ip +iputils-ping6 +iputils-ping \
-		+sprunge +safe-reboot +netperf +pv +tcpdump-mini +bwm-ng
+       		+busybox +ethtool +iwinfo +iw +mtr +ip +iputils-ping6 +iputils-ping \
+		+sprunge +safe-reboot +netperf +pv +tcpdump-mini \
+		+bwm-ng
```

Comment:
This doesn't come from me, it's from the revert of a Pau's commit 3e3c0f8 "Move lime-example", if you think it's better, I can avoid reverting it and build my PR on top of it.